CSharp/DevVmPowershell/Helpers/Implementations/ProcessingHelper.cs

```diff
@@ -111,59 +113,49 @@ public async Task<bool> FullResetAsync()
-				Order = 1
-			};
-
-			kCura.Relativity.Client.TextCondition cond = new kCura.Relativity.Client.TextCondition()
-			{
-				Field = Constants.Processing.NameField,
-				Operator = kCura.Relativity.Client.TextConditionEnum.EqualTo,
-				Value = Constants.Processing.ChoiceName
-			};
+			string url = Constants.Connection.RestUrlEndpoints.ProcessingManager.ProcessingSourceCreateUrl;
+			HttpClient httpClient = RestHelper.GetHttpClient(InstanceAddress, AdminUsername, AdminPassword);
 
-			using (IRSAPIClient rsapiClient = ServiceFactory.CreateProxy<IRSAPIClient>())
+			var createPayloadObject = new
 			{
-				rsapiClient.APIOptions = new APIOptions(-1);
-
-				kCura.Relativity.Client.DTOs.QueryResultSet<Choice> choiceQuery = rsapiClient.Repositories.Choice.Query(new Query<Choice>() { Condition = cond });
 
-				if (choiceQuery.Success && choiceQuery.TotalCount > 0)
+				choiceRequest = new
 				{
-					Console.WriteLine($"{nameof(CreateProcessingSourceLocationChoice)} - Failed to create Processing Source Location Choice ({Constants.Processing.ChoiceName}) as it already exists");
-					wasChoiceCreated = true;
-					return wasChoiceCreated;
-				}
+					Field = new { Guids = new List<Guid>{Guid.Parse(Constants.Processing.ProcessingSourceLocationFieldGuid)}},
+					Name = Constants.Processing.ChoiceName,
+					Order = 100,
+					Keywords = "DevVM Processing Source Location",
+					Notes = "",
+					RelativityApplications = new string[] { },
+			}
+			};
 
-				WriteResultSet<Choice> result = rsapiClient.Repositories.Choice.Create(choice);
+		    string createPayload = JsonConvert.SerializeObject(createPayloadObject);
+		    HttpResponseMessage createResponse = await RestHelper.MakePostAsync(httpClient, $"{Constants.Connection.RestUrlEndpoints.ProcessingManager.ProcessingSourceCreateUrl}/", createPayload);
+		    
+		    if (!createResponse.IsSuccessStatusCode)
+		    {
+			    throw new System.Exception("Failed to Create Processing Source Location");
+		    }
 
-				if (result.Success)
-				{
-					choiceQuery = rsapiClient.Repositories.Choice.Query(new Query<Choice>() { Condition = cond });
-					if (choiceQuery.Success && choiceQuery.TotalCount > 0)
-					{
-						wasChoiceCreated = true;
-						Console.WriteLine($"{nameof(CreateProcessingSourceLocationChoice)} - Successfully created Processing Source Location Choice ({Constants.Processing.ChoiceName})");
-					}
-					else
-					{
-						Console.WriteLine($"{nameof(CreateProcessingSourceLocationChoice)} - Failed to create Processing Source Location Choice ({Constants.Processing.ChoiceName})");
-					}
+			string resultString = await createResponse.Content.ReadAsStringAsync();
 
-				}
-				else
-				{
-					Console.WriteLine($"{nameof(CreateProcessingSourceLocationChoice)} - Failed to create Processing Source Location Choice ({Constants.Processing.ChoiceName})");
-				}
+			int processingChoiceId = Int32.Parse(resultString);
+
+			if (processingChoiceId == null || processingChoiceId < 0)

```

Comment:
processingChoiceId == null check can be removed as int doesn't allow null values.